static/js/types/modal_state.ts

```diff
@@ -43,7 +43,7 @@ abstract class ModalStateVariant<T> {
  * related to the content being edited.
  */
 export class Editing<T> extends ModalStateVariant<T> {
-  state: "editing" = "editing"
+  state = "editing" as const
```

Comment:
Apparently one of the linting packages had a breaking change in one of their non-major releases, which caused [`prefer-as-const`](https://typescript-eslint.io/rules/prefer-as-const/) to start taking effect.